Backticks have to be escaped in HEREDOCs. Remove it since it is ugly anyway.
Thanks Andrey Smachev.

Index: checkinstall/installwatch/installwatch
===================================================================
--- checkinstall.orig/installwatch/installwatch	2010-04-15 14:04:53.000000000 -0400
+++ checkinstall/installwatch/installwatch	2010-04-15 14:38:45.000000000 -0400
@@ -179,7 +179,7 @@
 		--copyright|--version)
 			cat << EOF
 
-Copyright (C) 1998 Pancrazio `Ezio' de Mauro 
+Copyright (C) 1998 Pancrazio 'Ezio' de Mauro
 Copyright (C) 2002 Felipe Eduardo Sanchez Diaz Duran <izto@asic-linux.com.mx>
 
     This program is free software; you can redistribute it and/or modify
